src/pages/dataset/[accession_id].astro

```diff
@@ -97,11 +101,16 @@ accession_id in StudyMetadata
             <DatasetInfo eic={eic_data} />
           ))
         }
+        {
```

Comment:
There's a bit of a mismatch in the mapping here with the annotation component. The overview is mapped to protocol description, leaving the Description field on the webpage empty.  
More models related but the models merge the "Annotation method" field with the description and I think we've discussed this a bit before but not sure this is the way to go. Also export related but the method_type seems to be exported as "other" as opposed to getting the info from. "Annotation Type" field.

Reply:
> Also export related but the method_type seems to be exported as "other" as opposed to getting the info from. "Annotation Type" field.

That's an issue with the ingestion as well - the model needs updating to deal with it. Currently the model only has 1 value but there are lists present.